apps/src/templates/codeDocs/CodeDocLink.jsx

```diff
@@ -12,7 +13,9 @@ export default function CodeDocLink({programmingExpression}) {
     );
   } else {
     return (
-      <a href={programmingExpression.link}>{programmingExpression.name}</a>
+      <Link href={programmingExpression.link} weight="regular">
```

Comment:
"regular" is the default for the `weight` prop, so you can remove it unless you want to make sure the weight is never affected